src/Storages/MergeTree/ReplicatedMergeTreeSink.cpp

```diff
@@ -464,7 +469,7 @@ void ReplicatedMergeTreeSink::commitPart(
                     else
                         quorum_path = storage.zookeeper_path + "/quorum/status";
 
-                    waitForQuorum(zookeeper, existing_part_name, quorum_path, quorum_info.is_active_node_value, replicas_num);
+                    waitForQuorum(zookeeper, existing_part_name, quorum_path, quorum_info.is_active_node_version, replicas_num);
```

Comment:
Why do we do this change? Is `is_active_node_value` still used?

Reply:
> Why do we do this change?

See https://github.com/ClickHouse/ClickHouse/pull/42541#issuecomment-1297273046

> Is is_active_node_value still used?

No, will remove it